src/signals/incident-management/components/CheckboxList/index.js

```diff
@@ -200,8 +200,8 @@ const CheckboxList = ({
         setToggled(allOptionsChecked);
 
         // in case that a list of options contains of only one item, we need to call the `onToggle`
-        // callback function instead of the `onChange` callback
-        if (allOptionsChecked) {
+        // callback function instead of the `onChange` callback unless the `onToggle` is undefined
+        if (onToggle && allOptionsChecked) {
```

Comment:
I don't see a test that covers this change. Can you confirm?

Reply:
Yes, and I have been thinking about it, let me ping you on Slack ;)